managers/bosh-manager/BoshTaskPoller.js

```diff
@@ -29,6 +30,10 @@ class BoshTaskPoller {
           const options = _.get(resourceBody, 'spec.options');
           const pollerAnnotation = _.get(resourceBody, 'metadata.annotations.lockedByTaskPoller');
           logger.debug(`pollerAnnotation is ${pollerAnnotation} current time is: ${new Date()}`);
+          let lastOperationOfInstance = {
+            state: 'in progress',
```

Comment:
isn't there a constant for `in progress` ?

Reply:
Actually it is state of osb api. So state should be 'in progress'.

Reply:
ok , fine.